pkg/kubernetes/client/client.go

```diff
@@ -178,23 +200,28 @@ func (cl *Client) CreateJivaVolume(req *csi.CreateVolumeRequest) error {
 	}
```

Comment:
Lets return error status as internal over here with error message

Reply:
Thanks for pointing this out.